kdevnel/ynab-absa-csv-convert-js
# Integrate Webpack

closes #12 

## Description
Sets up NPM and Webpack to do the following:

- The app has a development environment with dev dependencies
- The app is now set to bundle from /src into the /dist folder
- The /dist folder is emptied and the index.html file is dynamically generated using `html-webpack-plugin`
- Sets up `webpack-dev-server` for live preview of changes when `npm start` is used
- Dev environment is setup to use `inline-source-maps` (to be replaced with a more efficient approach in production)

## Testing
- Download the source folder and run `npm init` from the root of the project
- Run `npm start` to spin up the `webpack-dev-server` and open the site in your browser
- The app should function normally and code changes should be bundled and reloaded in real time.
